examples/README.md

```diff
@@ -0,0 +1,7 @@
+# Examples
```

Comment:
The main README I think should have a link to this folder, together with an explanation of what the purpose of these examples is and how they relate to the dev docs.